### PR TITLE
[CLI] Multi execution scanner and fixer

### PR DESCRIPTION
**Pull request by mantas-sidlauskas**
**What changed?**
Scanner/fixer part of CLI now works with a list of executions instead of list of shards. 
Adding or modifying execution scanners should not require any changes on CLI part, as scan types and invariants are set up using enumerators.
Some smaller changes:
 * Changed CLI panic to printing to stderr
 * added `github.com/dmarkham/enumer` to generate utility methods for const enums.
 * renamed `InvariantType` to `InvariantName`

**Why?**
To do system-wide scans we use cadence workflow, CLI tool is intended to troubleshoot and fix some specific executions.

**How did you test it?**
Unit tests

**Potential risks**

### CODE REVIEW

File: tools/cli/admin.go [hunk 1]
```diff
@@ -767,108 +771,53 @@ func newAdminQueueCommands() []cli.Command {
...
+
+	scanFlag := cli.StringFlag{
+		Name:     FlagScanType,
+		Usage:    "Scan type to use: " + strings.Join(executions.ScanTypeStrings(), ", "),
+		Required: true,
+	}
+
+	collectionsFlag := cli.StringSliceFlag{
+		Name:  FlagInvariantCollection,
+		Usage: "Scan collection type to use: " + strings.Join(collections, ", "),
```
Comment (vancexu) [inline tools/cli/admin.go:784]: Maybe add some example collection type here?

File: tools/cli/admin.go [hunk 2]
```diff
@@ -767,108 +771,53 @@ func newAdminQueueCommands() []cli.Command {
 }
 
 func newDBCommands() []cli.Command {
+	var collections cli.StringSlice = invariant.CollectionStrings()
+
+	scanFlag := cli.StringFlag{
+		Name:     FlagScanType,
+		Usage:    "Scan type to use: " + strings.Join(executions.ScanTypeStrings(), ", "),
```
Comment (vancexu) [inline tools/cli/admin.go:778]: Maybe add some examples type here?

**Review by yux0 — APPROVED** [review]

**Review by vancexu — APPROVED** [review]

File: cmd/tools/cli/main.go [hunk 3]
```diff
@@ -30,8 +31,8 @@ import (
 // See cadence/tools/cli/README.md for usage
 func main() {
 	app := cli.NewCliApp()
-	err := app.Run(os.Args)
-	if err != nil {
-		panic(err)
+	if err := app.Run(os.Args); err != nil {
+		fmt.Fprintln(os.Stderr, err.Error())
```
[the remark this answers is not in the document]
Comment (mantas-sidlauskas) [reply cmd/tools/cli/main.go:35]: I was considering to use `ErrorAndExit`, but:
* `ErrorAndExit` doesn't print to STDERR
* has features like stack printing and color output which are not needed here

File: tools/cli/admin.go (hunk 1 above)
Comment (mantas-sidlauskas) [reply tools/cli/admin.go:784]: All available collections will be printed out here, like this:
```
  --invariant_collection value    Scan collection type to use: CollectionMutableState, CollectionHistory (default: "CollectionMutableState", "CollectionHistory")
```

File: tools/cli/admin.go (hunk 2 above)
Comment (mantas-sidlauskas) [reply tools/cli/admin.go:778]: same as above

**Pull request merged**